internal/usage/usage.go

```diff
@@ -22,6 +22,7 @@ const (
 	Members                      = "Number of members in the replica set."
 	Shards                       = "Number of shards in the cluster."
 	ProcessName                  = "Unique identifier for the host of a MongoDB process in the following format: {hostname}:{port}."
+	ProcessNameAtlasCLI          = "Unique identifier for the host of a MongoDB process in the following format: {hostname}:{port}. List of possible values can be obtained by running: atlas processes list | jq '.results[].id'"
```

Comment:
can you please rever the `jq` part, it assumes you are using json as output which is not the default

Reply:
Would you preffer json-path or no outputs flags?

Reply:
no output